### DISCUSSION TITLE
v1.0.4

### DISCUSSION THREAD

**Discussion by Zeal-L** [Zeal-L/BiliBili-Manga-Downloader#26] [Announcements]
![](https://img.shields.io/github/downloads/Zeal-L/BiliBili-Manga-Downloader/v1.0.4/total)
- 优化使用体验：我的库存列表现在按照漫画名排序
- 优化项目结构: 重新分类了原始UI文件和资源文件，并更新了打包脚本
- 更新依赖项：更新了过去一个月积攒的 pyside6，pillow 等 python 库的新版本

## What's Changed
* Ready for 1.0.4 by @Zeal-L in https://github.com/Zeal-L/BiliBili-Manga-Downloader/pull/25

**Full Changelog**: https://github.com/Zeal-L/BiliBili-Manga-Downloader/compare/v1.0.3...v1.0.4

<hr /><em>This discussion was created from the release <a href='https://github.com/Zeal-L/BiliBili-Manga-Downloader/releases/tag/v1.0.4'>v1.0.4</a>.</em>

**Comment by Zeal-L** [comment]
时隔一个学期的更新~😄

**Comment by NOwin111** [comment]
很好用
